scaleway/resource_object_bucket_lock_configuration_test.go

```diff
@@ -0,0 +1,238 @@
+package scaleway
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	sdkacctest "github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+const (
+	LockResourcePrefix   = "tf-acc-test"
+	lockResourceTestName = "scaleway_object_bucket_lock_configuration.test"
+)
+
+func TestAccObjectBucketLockConfiguration_basic(t *testing.T) {
```

Comment:
Tests naming does not follow naming of others. Cassettes don't have resource name.